docs/topics/templates/document-attributes.adoc

```diff
@@ -13,19 +13,19 @@
 ifdef::mta[]
 :ProductName: Migration Toolkit for Applications
 :DocInfoProductName: Migration Toolkit for Applications
-:DocInfoProductNumber: 6.0
+:DocInfoProductNumber: 6.1
 :ProductShortName: MTA
 :LC_PSN: mta
 :DocInfoProductNameURL: migration_toolkit_for_applications
 :WebName: user interface
 :WebNameTitle: User Interface
 :WebNameURL: user_interface_guide
 :WebConsoleBookName: {WebNameTitle} Guide
-:ProductVersion: 6.0.1
+:ProductVersion: 6.1.0
 :PluginName: MTA plugin
-:MavenProductVersion: 6.0.1.GA-redhat-00006
-:ProductDistributionVersion: 6.0.1.GA-redhat
-:ProductDistribution: mta-6.0.1.GA-offline.zip
+:MavenProductVersion: 6.1.0.GA-redhat
+:ProductDistributionVersion: 6.1.0.GA-redhat
+:ProductDistribution: mta-6.1.0.GA-offline.zip
```

Comment:
@sshveta - again the you sent did not look like it was correct.

Please can I ask you to double-check before I do anything further?

Thanks